.readthedocs.yml

```diff
@@ -0,0 +1,20 @@
+# .readthedocs.yml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: doc/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: all
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.6
+  install:
+    - requirements: doc/requirements.txt
```

Comment:
Maybe I'm misunderstanding this, but how come these requirements don't include things like `sphinx` and `sphinx_rtd_theme`. Should I be able to create a Conda environment from this file alone and build the documentation?

Reply:
While on this point, why are those requirements included in the main `requirements.txt` anyway?

Reply:
No, `doc/requirements.txt` is a requirements file that is specific for ReadTheDocs build which uses a docker image that already includes sphinx, numpy, scipy etc which is why we don't need them. 

To do local development (whether code or documentation), you need to use the main `requirements.txt` file which installs everything you need. 

Good question. I will add this to the contributing section.